### DISCUSSION TITLE
MutableCompoundShape

### DISCUSSION THREAD

**Discussion by DennisSmolek** [jrouwe/JoltPhysics.js#170] [Q&A]
Is there a reason this isn't in the JS release or just not been asked for yet?

I'm being lazy and not just PR'ng a build myself but figured I'd ask if there was a specific reason for it. 

I want to be able to create custom bodies and update the shapes at runtime and this seems to be the method.

**Comment by jrouwe** [comment]
There's no other reason than that I just forgot to add it. It's added now in fcf5f5dfe2cc43428b8fb9778ae9208ecb339a5b.
